src/content/plugins/normal-module-replacement-plugin.md

```diff
@@ -40,50 +41,78 @@ Conditional build depending on an [specified environment](/configuration/configu
 Say you want a configuration with specific values for different build targets.
 
 ``` javascript
-module.exports = function(env) {
-  var appTarget = env.APP_TARGET || 'VERSION_A';
+module.exports = function(env, args) {
   return {
     plugins: [
       new webpack.NormalModuleReplacementPlugin(/(.*)-APP_TARGET(\.*)/, function(resource) {
-        resource.request = resource.request.replace(/-APP_TARGET/, `-${appTarget}`);
+        var original = resource.request;
+        var replace = resource.request.replace(/-APP_TARGET/, `-${env}`);
```

Comment:
Shouldn't this be `-${env.APP_TARGET}`?

Reply:
we would probably need to return the target declaration with fall back e.g. `const target = env || 'BETA';`
```suggestion
         var target = env || 'BETA';
         var replace = resource.request.replace(/-APP_TARGET/, `-${target}`);
```